generator/templates/server/parameter.gotmpl

```diff
@@ -191,7 +191,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}Params) BindRequest(r *http.Requ
     }
     {{ end }}
   {{ end }}} else {
-    {{ if .IsArray }}{{ if .Child }}{{ if (and (not .Schema.IsInterface) (or .Child.IsAliased .Child.IsComplexObject)) }}for _, {{ .IndexVar }}{{ .ReceiverName }} := range {{ .ReceiverName }}.{{ pascalize .Name }} {
+    {{ if .IsArray }}{{ if .Child }}{{ if (and (not .Schema.IsInterface) (or .Child.IsAliased .Child.IsComplexObject)) }}for _, {{ .IndexVar }}{{ .ReceiverName }} := range {{ if and (not .Schema.IsBaseType) .IsNullable }}&{{ end }}body {
```

Comment:
I think that just body would be enough, it won't hurt because when IsArray returns true IsNullable will always be false

so
```
{{ if and (not .Schema.IsBaseType) .IsNullable }}&{{ end }}
```

is not needed